src/htmlgen.cpp

```diff
@@ -1132,13 +1132,17 @@ void HtmlGenerator::writeStyleSheetFile(TextStream &t)
 
 void HtmlGenerator::writeHeaderFile(TextStream &t, const QCString & /*cssname*/)
 {
-  t << "<!-- HTML header for doxygen " << getDoxygenVersion() << "-->\n";
-  t << ResourceMgr::instance().getAsString("header.html");
+  QCString head =  ResourceMgr::instance().getAsString("header.html");
+  int endPlace = head.find("<head>") + 7;
```

Comment:
This look a bit hackish to me. It depends on header.html containing `<head>` and where is the magic number 7 coming from when `<head>` is only 6 characters? To skip over an assumed newline?

Maybe better to just add the comment after the header?

Reply:
I thought about the comment after the header but it would be far down and land after the comment `<!-- end header part -->`.
The header.html is the default doxygen header and this would, I think always, contain the `<head>`  tag on a single line. Hence also the magic number 7 i.e. the `<head>` plus the `\n`.

I can however quite easily change the pull request so the line is a the end, please advice.